File: Trivia/SendResultsResponse.cpp
#include "SendResultsResponse.h"

SendResultsResponse::SendResultsResponse(unsigned int position, std::vector<PlayerResult> results) : _position(position), _results(results)
{
}

unsigned int SendResultsResponse::getPosition() const
{
	return _position;
}

std::vector<PlayerResult> SendResultsResponse::getResults() const
{
	return _results;
}

File: TriviaClient/Responses/SendResultsResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TriviaClient.Responses
{
    class SendResultsResponse
    {
        int _position;
        string _winner;
        SendResultsResponse(int position, string winner)
        {
            _position = position;
            _winner = winner;
        }

        public int GetPosition()
        {
            return _position;
        }

        public string GetWinner()
        {
            return _winner;
        }
    }
}
